### PR TITLE
Object.xml -- updating for Try .NET

### PR DESCRIPTION
**Pull request by WilliamAntonRohm**
associated samples PR is https://github.com/dotnet/samples/pull/1640

### CODE REVIEW

File: xml/System/Object.xml [hunk 1]
```diff
@@ -198,7 +200,7 @@
   
     -   The two objects are of the same type. As the following example shows, a <xref:System.Byte> object that has a value of 12 does not equal an <xref:System.Int32> object that has a value of 12, because the two objects have different run-time types.  
   
-         [!code-csharp[System.Object.Equals#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equals_val1.cs#3)]
+         [!code-csharp-interactive[System.Object.Equals#3](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equals_val1.cs#3)]
```
Comment (mairaw) [inline xml/System/Object.xml]: this page then it's missing the note. add it to the remarks of the membergroup for Equals

File: xml/System/Object.xml [hunk 2]
```diff
@@ -219,7 +221,7 @@
   
  The following example provides an illustration. It instantiates three <xref:System.Text.StringBuilder> objects with identical strings, and then makes four calls to `Equals` methods. The first method call returns `true`, and the remaining three return `false`.  
   
- [!code-csharp[System.Object.Equals#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs#5)]
+ [!code-csharp-interactive[System.Object.Equals#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs#5)]
```
Comment (mairaw) [inline xml/System/Object.xml]: we undid this change

```suggestion
 [!code-csharp[System.Object.Equals#5](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.equals/cs/equalssb1.cs#5)]
```

File: xml/System/Object.xml [hunk 3]
```diff
@@ -762,15 +764,15 @@ and the <xref:System.IDisposable> interface. The <xref:System.IDisposable.Dispos
   
  For two objects `x` and `y` that have identical runtime types, `Object.ReferenceEquals(x.GetType(),y.GetType())` returns `true`. The following example uses the <xref:System.Object.GetType%2A> method with the <xref:System.Object.ReferenceEquals%2A> method to determine whether one numeric value is the same type as two other numeric values.
```
Comment (mairaw) [inline xml/System/Object.xml]: this page is also missing the interactive note

File: xml/System/Object.xml [hunk 4]
```diff
@@ -922,14 +924,14 @@ and the <xref:System.IDisposable> interface. The <xref:System.IDisposable.Dispos
   
 -   When comparing value types. If `objA` and `objB` are value types, they are boxed before they are passed to the <xref:System.Object.ReferenceEquals%2A> method. This means that if both `objA` and `objB` represent the same instance of a value type, the <xref:System.Object.ReferenceEquals%2A> method nevertheless returns `false`, as the following example shows.
```
Comment (mairaw) [inline xml/System/Object.xml]: missing the interactive note as well

File: xml/System/Object.xml [hunk 5]
```diff
@@ -1008,7 +1010,7 @@ and the <xref:System.IDisposable> interface. The <xref:System.IDisposable.Dispos
  The default implementation of the <xref:System.Object.ToString%2A> method returns the fully qualified name of the type of the <xref:System.Object>, as the following example shows.  
   
  [!code-cpp[System.Object.ToString#1](~/samples/snippets/cpp/VS_Snippets_CLR_System/system.object.tostring/cpp/tostring1.cpp#1)]
- [!code-csharp[System.Object.ToString#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.tostring/cs/tostring1.cs#1)]
+ [!code-csharp-interactive[System.Object.ToString#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.object.tostring/cs/tostring1.cs#1)]
```
Comment (mairaw) [inline xml/System/Object.xml]: this page is also missing the interactive note

File: xml/System/Object.xml (hunk 1 above)
Comment (WilliamAntonRohm) [reply xml/System/Object.xml]: Done.

File: xml/System/Object.xml [hunk 6]
```diff
@@ -70,6 +70,8 @@
 ## Examples  
  The following example defines a Point type derived from the <xref:System.Object> class and overrides many of the virtual methods of the <xref:System.Object> class. In addition, the example shows how to call many of the static and instance methods of the <xref:System.Object> class.  
   
+ [!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```
[the remark this answers is not in the document]
Comment (WilliamAntonRohm) [reply xml/System/Object.xml]: Removed.

File: xml/System/Object.xml (hunk 2 above)
Comment (WilliamAntonRohm) [reply xml/System/Object.xml]: Updated.

File: xml/System/Object.xml (hunk 4 above)
Comment (WilliamAntonRohm) [reply xml/System/Object.xml]: Added.

File: xml/System/Object.xml (hunk 3 above)
Comment (WilliamAntonRohm) [reply xml/System/Object.xml]: Added.

File: xml/System/Object.xml (hunk 5 above)
Comment (WilliamAntonRohm) [reply xml/System/Object.xml]: Added.

**Comment by WilliamAntonRohm** [comment]
@mairaw -- Maira, I believe I've addressed all your review comments.  Thank you for your continued assistance.

**Comment by WilliamAntonRohm** [comment]
#sign-off

**Comment by mairaw** [comment]
Resolved some merge conflicts

**Comment by mairaw** [comment]
OPS build finished successfully so merging this.

**Pull request merged**